test/extended/cluster/context.go

```diff
@@ -99,13 +99,13 @@ type ServiceInfo struct {
 // ParseConfig will complete flag parsing as well as viper tasks
 func ParseConfig(config string, isFixture bool) error {
 	// This must be done after common flags are registered, since Viper is a flag option.
-	if isFixture {
+	if filepath.IsAbs(config) || isFixture {
 		dir, file := filepath.Split(config)
 		s := strings.Split(file, ".")
```

Comment:
Lack of consistency here to remove the extension if passed, as this `strings.Split` function is faster but doesn't work for the relative path condition.